daemons/dnssec/ipa-dnskeysync-replica.in

```diff
@@ -145,7 +146,11 @@ def ldap2replica_zone_keys_sync(ldapkeydb, localhsm):
 
 
 # IPA framework initialization
-standard_logging_setup(debug=True)
+debug = get_config_debug('dns')
+standard_logging_setup(debug=debug)
+if not debug:
```

Comment:
By default, only errors messages are logged to the console: https://github.com/freeipa/freeipa/blob/ce0592bd478c184b1c0c2a75f675c5a361f4a9a4/ipapython/ipa_log_manager.py#L92-L99
To have logger.info write in the console, `standard_logging_setup` has to be called with `(debug=debug, verbose=True)`.

Reply:
@rcritten could you please update this too?

Reply:
Sorry, I missed this comment. The debug level is higher than info so all message should be logged. I don't believe we need to set verbose as debug does the same thing.

Reply:
The intent of line 152 was to print the guidance in all cases (even if debug is not set), but it's using `logger.info`. If debug is not set in the `dns.conf` file, then the console prints only ERROR messages and the INFO message `To increase debugging...` never goes to the journal.